<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>221</width>
      <height>621</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(222, 222, 222);
</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>160</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_pageSelector_environment">
           <property name="text">
            <string>Environment</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>170</y>
          <width>160</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>200</y>
          <width>160</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3"/>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>230</y>
          <width>160</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4"/>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>160</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5"/>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>66</width>
          <height>66</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="bbqlinux_logo">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="qt_resources.qrc">:/icons/bbqlinux_icon_blue_64x64.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_13">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Sans</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>Control Panel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>571</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QStackedWidget" name="pageStack">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page_welcome"/>
       <widget class="QWidget" name="page_java">
        <widget class="QWidget" name="verticalLayoutWidget_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>551</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_java_4">
            <property name="font">
             <font>
              <family>Droid Sans</family>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Environment</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_8">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>371</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_java">
            <property name="font">
             <font>
              <family>Droid Sans</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Java version</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_9">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>371</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QComboBox" name="comboBox_java">
            <property name="editable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_10">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>130</y>
           <width>371</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_python">
            <property name="font">
             <font>
              <family>Droid Sans</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Python version</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_11">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>371</width>
           <height>21</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11"/>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_12">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>371</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QComboBox" name="comboBox_python"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="page_python">
        <widget class="QPushButton" name="button_python3">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>190</y>
           <width>91</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Active</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_python2">
         <property name="geometry">
          <rect>
           <x>204</x>
           <y>160</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Python 2</string>
         </property>
        </widget>
        <widget class="QPushButton" name="button_python2">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>190</y>
           <width>91</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Active</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_python3">
         <property name="geometry">
          <rect>
           <x>354</x>
           <y>160</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Python 3</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_quit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>580</y>
      <width>79</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="qt_resources.qrc"/>
 </resources>
 <connections/>
</ui>
